Cesium3DTilesSelection/src/TilesetContentManager.cpp

```diff
@@ -219,6 +219,10 @@ void TilesetContentManager::updateTileContent(Tile& tile) {
   default:
     break;
   }
+
+  if (content.shouldContentContinueUpdated()) {
+    content.setContentShouldContinueUpdated(_pLoader->updateTileContent(tile));
```

Comment:
I think it's also necessary to defer transitioning the Tile to the Done state until `updateTileContent` gives the OK. Or at least, we'll need some way to make sure we don't create upsampled children before the real children have been created.

Reply:
I can see it now. I think it maybe better to solve the transition in the raster overlay PR